docs/backtesting.md

```diff
@@ -1,88 +1,100 @@
 # Backtesting
+
 This page explains how to validate your strategy performance by using 
 Backtesting.
 
 ## Table of Contents
+
 - [Test your strategy with Backtesting](#test-your-strategy-with-backtesting)
 - [Understand the backtesting result](#understand-the-backtesting-result)
 
 ## Test your strategy with Backtesting
+
 Now you have good Buy and Sell strategies, you want to test it against
 real data. This is what we call 
 [backtesting](https://en.wikipedia.org/wiki/Backtesting).
 
-
 Backtesting will use the crypto-currencies (pair) from your config file
 and load static tickers located in 
 [/freqtrade/tests/testdata](https://github.com/freqtrade/freqtrade/tree/develop/freqtrade/tests/testdata).  
 If the 5 min and 1 min ticker for the crypto-currencies to test is not 
 already in the `testdata` folder, backtesting will download them 
 automatically. Testdata files will not be updated until you specify it.
 
-The result of backtesting will confirm you if your bot as more chance to
-make a profit than a loss.
-
+The result of backtesting will confirm you if your bot as more chance to make a profit than a loss.
```

Comment:
as = has. and "more chance" doesn't sound like english either. "better odds of making a profit than a loss" maybe..?

Reply:
nice catch - i guess if english language is the problem in the PR it could be worse ... 
especially since i didn't really read this part of the doc - but only removed the newline